src/components/marketDetailPage/Menu.style.tsx

```diff
@@ -77,7 +77,7 @@ const MenuCounterButtonWrapper = styled.TouchableOpacity`
   justify-content: center;
   align-items: center;
   width: 32px;
-  background-color: lightblue;
+  background-color: #70c802;
```

Comment:
```suggestion
  background-color: ${props => props.theme.colors.primary}
```

theme를 적용할 수 있으면 적용해주세요! `#70c802` hex code와 동일한 색깔입니다.

ref] [feat: 테마 적용](#58)